### PR TITLE
RED-428 removed sensitivity to the case

### PR DESCRIPTION
**Pull request by au-e2xlabs**

### CODE REVIEW

File: src/redefined.resolver.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export class RedefinedResolver implements Resolver {
         // throwErrorOnInvalidDomain - non-rewritable parameter
         // If it is true, then everything will fall on an error in one resolver
         return flatten(
-          await Promise.all(this.resolvers.map(it => it.resolve(domain, { ...customOptions, throwErrorOnInvalidDomain: false }, networks)))
+          await Promise.all(this.resolvers.map(it => it.resolve(domain.toLowerCase(), { ...customOptions, throwErrorOnInvalidDomain: false }, networks)))
```
Comment (Ivan-dev-RED) [inline src/redefined.resolver.ts:31]: I think this fix is not needed fot JS lib
Only backend and connect-web
But it needs to be clarified

Comment (rnovokhatsky) [reply src/redefined.resolver.ts:31]: why not? but I believe lowercase should happen in RedefinedEmailResolverService and RedefinedNicknameResolverService

Comment (Ivan-dev-RED) [reply src/redefined.resolver.ts:31]: @rnovokhatsky 
May be only in these services
BUT is it correct for ENS and Uns?

Comment (rnovokhatsky) [reply src/redefined.resolver.ts:31]: ENS and Uns encapsulate their own logic, we should not bother

Comment (Ivan-dev-RED) [reply src/redefined.resolver.ts:31]: @au-e2xlabs 
So, need to transfer this code to `redefined-resolver.service.ts`

**Review by Ivan-dev-RED — APPROVED** [review]

**Pull request merged**